src/routes/auth/auth.controller.ts

```diff
@@ -19,8 +28,19 @@ import { getMillisecondsUntil } from '@/domain/common/utils/time';
 @ApiExcludeController()
 export class AuthController {
   static readonly ACCESS_TOKEN_COOKIE_NAME = 'access_token';
+  static readonly ACCESS_TOKEN_COOKIE_SAME_SITE_LAX = 'lax';
+  static readonly ACCESS_TOKEN_COOKIE_SAME_SITE_NONE = 'none';
+  static readonly CGW_ENV_PRODUCTION = 'production';
+  private readonly cgwEnv: string;
```

Comment:
What do you think about adding an `isProduction` env. var. as well?

Reply:
Do you mean instead of this one, or in addition to it? (I'm not against it, just aligning regarding the use cases it would have)

Reply:
I think the ternary comparison could be done against this value _in_ the configuration, e.g. there is an `application.isProduction` value.

Reply:
It makes perfect sense, thanks! I've created a PR: https://github.com/safe-global/safe-client-gateway/pull/1724